streaming-binlog/src/main/java/org/openmrs/analytics/DebeziumListener.java

```diff
@@ -63,20 +64,24 @@ FhirConverter createFhirConverter(CamelContext camelContext) throws IOException,
 		ParquetUtil parquetUtil = new ParquetUtil(params.fileParquetPath, params.secondsToFlushParquetFiles,
 		        params.rowGroupSizeForParquetFiles);
 		camelContext.addService(new ParquetService(parquetUtil), true);
-		return new FhirConverter(openmrsUtil, fhirStoreUtil, parquetUtil, params.fhirDebeziumEventConfigPath);
+		return new FhirConverter(openmrsUtil, fhirStoreUtil, parquetUtil, params.fhirDebeziumConfigPath);
 	}
 	
-	private String getDebeziumConfig() {
-		return "debezium-mysql:" + params.databaseHostName + "?" + "databaseHostname=" + params.databaseHostName
-		        + "&databaseServerId=" + params.databaseServerId + "&databasePort=" + params.databasePort.intValue()
-		        + "&databaseUser=" + params.databaseUser + "&databasePassword=" + params.databasePassword
-				//+ "&name={{database.dbname}}"
-		        + "&databaseServerName=" + params.databaseName + "&databaseWhitelist=" + params.databaseSchema
-		        + "&offsetStorage=org.apache.kafka.connect.storage.FileOffsetBackingStore" + "&offsetStorageFileName="
-		        + params.databaseOffsetStorage + "&databaseHistoryFileFilename=" + params.databaseHistory + "&snapshotMode="
-		        + params.snapshotMode
-		//+ "&tableWhitelist={{database.schema}}.encounter,{{database.schema}}.obs"
-		;
+	private String getDebeziumConfig() throws IOException {
+		GeneralConfiguration gc = new GeneralConfiguration();
+		GeneralConfiguration generalConfiguration = gc.getFhirDebeziumConfigPath(params.fhirDebeziumConfigPath);
+		return "debezium-mysql:" + generalConfiguration.getDbzConfigs().get("databaseHostName") + "?" + "databaseHostname="
+		        + generalConfiguration.getDbzConfigs().get("databaseName") + "&databaseServerId="
+		        + generalConfiguration.getDbzConfigs().get("databaseServerId") + "&databasePort="
+		        + generalConfiguration.getDbzConfigs().get("databasePort") + "&databaseUser="
+		        + generalConfiguration.getDbzConfigs().get("databaseUser") + "&databasePassword="
+		        + generalConfiguration.getDbzConfigs().get("databasePassword") + "&databaseServerName="
+		        + generalConfiguration.getDbzConfigs().get("databaseName") + "&databaseWhitelist="
+		        + generalConfiguration.getDbzConfigs().get("databaseSchema") + "&offsetStorage="
```

Comment:
Are we expecting that we may need to change `offsetStorage` settings? If not, we should probably hard-code it here and not expose it in the config file.